cdp_scrapers/instances/seattle.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from ssl import get_protocol_name
+from typing import Dict, List
+from bs4 import BeautifulSoup
+from urllib.request import urlopen
+from urllib.error import HTTPError
+import re
+import sys
+import datetime
+
+from cdp_backend.pipeline.ingestion_models import (
+    EventIngestionModel,
+    Body,
+    Session,
+    EventMinutesItem,
+    MinutesItem,
+    Matter,
+    SupportingFile,
+    Person,
+    Vote,
+)
+from cdp_scrapers.legistar_utils import(
+    get_legistar_events_for_timespan,
+    LEGISTAR_BASE
+)
+
+###############################################################################
+
+
+# e.g. MinutesItem.name     =  EventItemEventId from legistar api
+LEGISTAR_MINUTE_NAME        = 'EventItemEventId'
+LEGISTAR_EV_MINUTE_DECISION = 'EventItemPassedFlagName'
+# TODO: is it better to use uuids like EventItemMatterGuid for cdp *_ext_src_id?
```

Comment:
Good question. Let's stick with IDs for now. I think you can directly query those documents with the actual ID and not the GUID.